docs/topics/loaders.rst

```diff
@@ -174,600 +88,31 @@ is an example::
 
         # ...
 
-As you can see, input processors are declared using the ``_in`` suffix while
-output processors are declared using the ``_out`` suffix. And you can also
-declare a default input/output processors using the
+In this case, field input and output processors are declared as the item's
+field name followed by a ``_in`` or ``_out`` suffix.
+If there's some routine that's generic and shareable across the fields,
+it can be define as a default processor through
 :attr:`ItemLoader.default_input_processor` and
 :attr:`ItemLoader.default_output_processor` attributes.
 
-.. _topics-loaders-processors-declaring:
-
-Declaring Input and Output Processors
-=====================================
-
-As seen in the previous section, input and output processors can be declared in
-the Item Loader definition, and it's very common to declare input processors
-this way. However, there is one more place where you can specify the input and
-output processors to use: in the :ref:`Item Field <topics-items-fields>`
-metadata. Here is an example::
```

Comment:
Is there an example of using Item meta for declaring of input and output processors in the new docs?